core/src/main/scala/org/apache/spark/scheduler/StageInfo.scala

```diff
@@ -61,6 +61,21 @@ class StageInfo(
       "running"
     }
   }
+
+  private[spark] def getStatusDetail: String = {
```

Comment:
Yeah, I mean I wonder if it's worth exposing this pretty specific logging string as an internal API method? you can just create this string in the one place it's used